internal/pkg/api/server/templates/extension/anytype/anytype.go

```diff
@@ -0,0 +1,56 @@
+// Package anytype contains extension to fix Goa bug: Any type is generated as "string:$binary" but the type should be empty/omitted.
+// See https://github.com/goadesign/goa/issues/3055
+package anytype
```

Comment:
Hodnota `user input` moze mat `Any` type, podla toho o aky vstup sa jedna:
https://github.com/keboola/keboola-as-code/blob/78d07ccdd1587a14a2899e690cbb51ff56ac6903/api/templates/design.go#L474-L485

Funguje to spravne, len je tam problem, ze Goa do OpenApi vygeneruje:
```
type: string
format: binary
```

Riesia to tu, zjavne to asi uplne nedomysleli:
https://github.com/goadesign/goa/blob/08a30a9374608d619d3ea8882cf77426071d2fe8/http/codegen/openapi/v3/types.go#L162-L171

Je na to nahlaseny bug: https://github.com/goadesign/goa/issues/3055